orientdb/src/main/java/com/yahoo/ycsb/db/OrientDBClient.java

```diff
@@ -68,42 +76,70 @@ public void init() throws DBException {
     String user = props.getProperty(USER_PROPERTY, USER_PROPERTY_DEFAULT);
     String password = props.getProperty(PASSWORD_PROPERTY, PASSWORD_PROPERTY_DEFAULT);
     Boolean newdb = Boolean.parseBoolean(props.getProperty(NEWDB_PROPERTY, NEWDB_PROPERTY_DEFAULT));
+    String remoteStorageType = props.getProperty(STORAGE_TYPE_PROPERTY);
 
     if (url == null) {
       throw new DBException(String.format("Required property \"%s\" missing for OrientDBClient", URL_PROPERTY));
     }
 
-    try {
-      System.out.println("OrientDB loading database url = " + url);
-
-      OGlobalConfiguration.STORAGE_KEEP_OPEN.setValue(false);
-      db = new ODatabaseDocumentTx(url);
-      if (db.exists()) {
-        db.open(user, password);
-        if (newdb) {
-          System.out.println("OrientDB drop and recreate fresh db");
-          db.drop();
-          db.create();
-        }
-      } else {
-        System.out.println("OrientDB database not found, create fresh db");
-        db.create();
-      }
-
-      System.out.println("OrientDB connection created with " + url);
+    System.out.println("OrientDB loading database url = " + url);
```

Comment:
Sure, I can do that for now. I have plans to move the whole binding to slf4j, which I think the entire project should move to, but it can be tackled in increments.